lib/generapp/actions/develop.rb

```diff
@@ -1,28 +1,19 @@
-require 'generapp/actions/files'
 
 module Generapp
-  module Actions
+  module Actions #:nodoc
+    # App develop environment
+    # associated actions
     module Develop
-      include Files
-
       def raise_on_delivery_errors
-        replace_in_file 'config/environments/development.rb',
-                        'raise_delivery_errors = false', 'raise_delivery_errors = true'
+        gsub_file 'config/environments/development.rb',
+                  'raise_delivery_errors = false',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.